client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
+		margin-top: 0;
+	}
+}
+
+.plan-thank-you-card__header {
+	background-color: $alert-green;
+	padding: 75px 0 30px;
+	position: relative;
+	z-index: 1;
+	line-height: 1.2;
+	border-radius: 6px 6px 0 0;
+}
+
+.plan-thank-you-card__body {
+	padding: 40px;
+}
+
+.plan-thank-you-card__heading {
+	font-size: 22px;
+	font-weight: 600;
+	margin-bottom: 8px;
+}
+
+.plan-thank-you-card__description {
+	font-size: 15px;
+	font-weight: 400;
+	width: 280px;
+	max-width: 100%;
+	margin: 0 auto 30px;
+	line-height: 20px;
+}
+
+.plan-thank-you-card__button {
+	display: inline-block;
+	border-radius: 4px;
+	background-color: $white;
+	color: darken( $alert-green, 30 );
+	font-size: 14px;
+	height: 40px;
+	line-height: 40px;
+	padding: 0 60px;
+	border-style: solid;
+	border-color: darken( $alert-green, 20 );
+	border-width: 1px 1px 2px;
+
+	//box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3),
```

Comment:
I should have reviewed the .scss file after an unnamed designer contributed there! 😼 Joking, he has done a great job making the component feel alive 👍